### DISCUSSION TITLE
Troubleshooting Artillery Sidewinder X2: Resuming Prints After Filament Runout

### DISCUSSION THREAD

**Discussion by sakomoto987** [OrcaSlicer/OrcaSlicer#2090] [Q&A] [answered]
Hello folks!

I've encountered a similar issue with all 5 of my Artillery Sidewinder printers. After reloading the filament following a filament runout, the print resumes, and X, Y, and Z movements seem fine. However, the extruder motor remains stationary, causing the filament not to be loaded and extruded. This results in the need to cancel the ongoing print, leading to both time and financial losses. Has anyone else experienced this problem?

Thanks

👀 @SoftFever
@fr3ak2402

**Comment by zetabemol** [comment, marked as answer]
Hola
En ajustes de la impresora/información básica debes **destildar** la opción "Usar distancias relativas en E"
En ajustes de la impresora/g-code de la máquina/g-code para antes del cambio de capa debes escribir ";" delante del "G92"
Saludos

**Reply by sakomoto987** [reply]
OMG ! It works at least! Thank you so much! You can't imagine how many spools I couldn't end and how many prints just failed..
Now I can use fully Orca, which will help me a lot in the production process. Gosh you made my day! I'll relay the solution everywhere I posted my issue.
Question, how did you find the fix? Experience? Same issue?
